apps/src/applab/DesignToolbox.jsx

```diff
@@ -19,6 +19,7 @@ export default class DesignToolbox extends React.Component {
         width: 270,
         boxSizing: 'border-box',
         borderRight: '1px solid gray',
+        overflowY: 'scroll',
```

Comment:
I think this might have caused the toolbox to only have one column:
![image](https://user-images.githubusercontent.com/4640747/32741888-9847ecc0-c85c-11e7-8fe7-b2423bccbcf7.png)